examples/basic/package-lock.json

```diff
@@ -15,6 +15,7 @@
     },
     "../../packages/backend-postgres": {
       "name": "@openworkflow/backend-postgres",
+      "version": "0.2.0",
```

Comment:
the changes to this file should be undone since no changes were made in this dir